UPGRADING.md

```diff
@@ -41,6 +41,19 @@ Prior to this version the response would be `one is missing`.
 
 See [#1510](https://github.com/ruby-grape/grape/pull/1510) for more information.
 
+#### Change default status code of DELETE to 204
```

Comment:
Should the change, so _The default status code for DELETE is now 204 instead of 200._